CHANGELOG.md

```diff
@@ -0,0 +1,50 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [2.0.0] - 2020-01-10
+### Added
+- Big changes in this release where our SDK has now been modularized:
+Each feature has its independent dependency therefore, integrators only need to include the dependencies of the features they are interested in.
```

Comment:
"independent dependency" - seems a bit of an oxymoron?  I know the word 'dependency' is being used in context of technical development, but we might want to think about rewording this.

Reply:
I have changed it to "its own dependency". 👍